### PR TITLE
[FEAT] Adding seed.bought event

### PR DESCRIPTION
**Pull request by HBandesh**
# Description

In this PR, I have added an event by which the user would be able to buy the seeds from market. This functionality will be guided by the level of the bumpkin.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -19,6 +19,10 @@
     filter: drop-shadow(1px 1px 0px white) drop-shadow(-1px 0px 0px white)
       drop-shadow(0px -1px 0px white);
   }
+
+  .seperate-lines {
```
Comment (mGranTY) [inline src/styles.css:23]: typo here should be `separate-lines`

Comment (mGranTY) [inline src/styles.css:23]: typo here should be `separate-lines`

File: src/features/game/events/seedBought.test.ts [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            experience: 100,
+          },
+        },
+        action: {
+          type: "seed.bought",
+          item: "Pumpkin Seed",
+          amount: 1,
+        },
+      })
+    ).toThrow("Inadequate level");
```
Comment (adamhannigan) [inline src/features/game/events/seedBought.test.ts:39]: Fancy and polite error messages 🏆

**Review by adamhannigan — DISMISSED** [review]
Awesome work, especially with the tests 🏆 

This looks good to me - we can pair today around how these analytics will work

**Review by adamhannigan — APPROVED** [review]
Awesome work! 🚀

**Pull request merged**